src/pages/ir-defaults/form/IrDefaultForm.js

```diff
@@ -67,6 +69,27 @@ const IrDefaultForm = ({ _labels, access }) => {
       <VertLayout>
         <Grow>
           <Grid container spacing={4}>
+            <Grid item xs={12}>
+              <ResourceComboBox
+                endpointId={SystemRepository.DocumentType.qry}
+                parameters={`_startAt=0&_pageSize=1000&_dgId=${SystemFunction.MaterialTransfer}`}
+                name='ir_tfr_DocTypeId'
+                required
```

Comment:
are u sure it's required ? it's a default combo i don't think that it's required